### PR TITLE
doc/cephadm: refine "Removing Hosts"

### PR DESCRIPTION
**Pull request by zdover23**
An intended edit to remove a redundant indefinite article became a longer (but still brief) full editorial pass.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/cephadm/host-management.rst [hunk 1]
```diff
@@ -88,23 +88,26 @@ To drain all daemons from a host, run a command of the following form:
 
    ceph orch host drain *<host>*
 
-The '_no_schedule' label will be applied to the host. See :ref:`cephadm-special-host-labels`
+The ``_no_schedule`` label will be applied to the host. See
+:ref:`cephadm-special-host-labels`.
 
-All osds on the host will be scheduled to be removed. You can check osd removal progress with the following:
+All osds on the host will be scheduled to be removed. You can check the progress of the osd removal operation with the following command:
```
Comment (anthonyeleven) [inline doc/cephadm/host-management.rst:94]: OSDs, "progress of OSD removal"

Comment (colemitchell) [reply doc/cephadm/host-management.rst:94]: @zdover23 I'm merging this now, but I recommend making _osd_ uppercase in a future PR.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/49706 - Quincy backport
https://github.com/ceph/ceph/pull/49707 - Pacific backport

File: doc/cephadm/host-management.rst (hunk 1 above)
Comment (zdover23) [reply doc/cephadm/host-management.rst:94]: "osd" is now "OSD" as of 15 Apr 2023.

Comment (zdover23) [reply doc/cephadm/host-management.rst:94]: https://github.com/ceph/ceph/pull/49715 - doc/cephadm: s/osd/OSD/ where appropriate

**Pull request merged**